tutorcredentials/plugin.py

```diff
@@ -19,78 +19,66 @@
+    },
+    "unique": {
+        "MYSQL_PASSWORD": "{{ 8|random_string }}",
+        "SOCIAL_AUTH_EDX_OAUTH2_SECRET": "{{ 16|random_string }}",
+        "BACKEND_SERVICE_EDX_OAUTH2_SECRET": "{{ 16|random_string }}",
+        "OAUTH2_SECRET": "{{ 16|random_string }}",
+        "OAUTH2_SECRET_DEV": "{{ 16|random_string }}",
+        "OAUTH2_SECRET_SSO": "{{ 16|random_string }}",
+        "OAUTH2_SECRET_SSO_DEV": "{{ 16|random_string }}",
+    },
+}
+
 tutor_hooks.Filters.CONFIG_DEFAULTS.add_items(
-    [
-        # Add your new settings that have default values here.
-        # Each new setting is a pair, (setting_name, default_value).
-        # Prefix your setting names with 'CREDENTIALS_'.
-        ("CREDENTIALS_VERSION", __version__),
-        ("CREDENTIALS_BACKEND_SERVICE_EDX_OAUTH2_PROVIDER_URL", "http://lms:8000/oauth2"),
-        ("CREDENTIALS_BACKEND_SERVICE_EDX_OAUTH2_KEY", "{{ CREDENTIALS_OAUTH2_KEY }}"),
-        ("CREDENTIALS_DOCKER_IMAGE", "{{ DOCKER_REGISTRY }}overhangio/openedx-credentials:{{ CREDENTIALS_VERSION }}"),
-        ("CREDENTIALS_EXTRA_PIP_REQUIREMENTS", []),
-        ("CREDENTIALS_FAVICON_URL", "https://edx-cdn.org/v3/default/favicon.ico"),
-        ("CREDENTIALS_HOST", "credentials.{{ LMS_HOST }}"),
-        ("CREDENTIALS_LOGO_TRADEMARK_URL", "https://edx-cdn.org/v3/default/logo-trademark.svg"),
-        ("CREDENTIALS_LOGO_TRADEMARK_URL_PNG", "https://edx-cdn.org/v3/default/logo-trademark.png"),
-        ("CREDENTIALS_LOGO_TRADEMARK_URL_SVG", "https://edx-cdn.org/v3/default/logo-trademark.svg"),
-        ("CREDENTIALS_LOGO_URL", ""),
-        ("CREDENTIALS_LOGO_URL_PNG", "{{ CREDENTIALS_LOGO_URL }}"),
-        ("CREDENTIALS_LOGO_URL_SVG", ""),
-        ("CREDENTIALS_LOGO_WHITE_URL", "{{ CREDENTIALS_LOGO_URL }}"),
-        ("CREDENTIALS_LOGO_WHITE_URL_PNG", "{{ CREDENTIALS_LOGO_URL }}"),
-        ("CREDENTIALS_LOGO_WHITE_URL_SVG", ""),
-        ("CREDENTIALS_MYSQL_DATABASE", "credentials"),
-        ("CREDENTIALS_MYSQL_USERNAME", "credentials"),
-        ("CREDENTIALS_OAUTH2_KEY", "credentials-key"),
-        ("CREDENTIALS_OAUTH2_KEY_DEV", "credentials-key-dev"),
-        ("CREDENTIALS_OAUTH2_KEY_SSO", "credentials-key-sso"),
-        ("CREDENTIALS_OAUTH2_KEY_SSO_DEV", "credentials-key-sso-dev"),
-        ("CREDENTIALS_PLATFORM_NAME", "{{ PLATFORM_NAME }}"),
-        ("CREDENTIALS_SITE_NAME", "{{ LMS_HOST }}"),
-        ("CREDENTIALS_SOCIAL_AUTH_REDIRECT_IS_HTTPS", False),
-        ("CREDENTIALS_SOCIAL_AUTH_EDX_OAUTH2_ISSUER", "https://{{ LMS_HOST }}"),
-        ("CREDENTIALS_SOCIAL_AUTH_EDX_OAUTH2_URL_ROOT", "http://lms:8000"),
-        ("CREDENTIALS_SOCIAL_AUTH_EDX_OAUTH2_KEY", "credentials-sso-key"),
-        ("CREDENTIALS_SOCIAL_AUTH_EDX_OAUTH2_LOGOUT_URL", "{{ LMS_HOST }}/logout"),
-        ("CREDENTIALS_THEME_NAME", "edx-theme"),
-        ("CREDENTIALS_REPOSITORY", "https://github.com/openedx/credentials.git"),
-        ("CREDENTIALS_REPOSITORY_VERSION", "{{ OPENEDX_COMMON_VERSION }}"),
```

Comment:
@CodeWithEmad you missed these two settings: CREDENTIALS_REPOSITORY, CREDENTIALS_REPOSITORY_VERSION

As a consequence, CI fails during `tutor config save`. Can you please fix this? cc @Faraz32123

Reply:
I'll fix it asap.

Reply:
fixed it.